.buildkite/pipeline.yml

```diff
@@ -8,8 +8,9 @@ env:
 
```

Comment:
`JULIA_VERSION`, `OPENMPI_VERSION`, `CUDA_VERSION` can all go. I think BUILDKITE_COMMIT and BUILDKITE_BRANCH can be removed as well.